doc/release-notes-26471.md

```diff
@@ -0,0 +1,14 @@
+Updated settings
+----------------
+
+- Previously running with the `-blocksonly` option effectively disabled the
+  mempool but continued to allocate 300MB of cache for it, which would be
+  shared with the dbcache. This might lead to unexpected resource usage of
+  nodes running with this option during for example IBD.
```

Comment:
```suggestion
  shared with the dbcache. This might lead to nodes running with this
  option using more resources than anticipated, especially during IBD.
```

Reply:
Done